kernel/src/kalloc.cpp

```diff
@@ -580,7 +580,7 @@ void kalloc::debug(){
 
     k_print_line();
 
-    printf("memory free in malloc chain: %m (%u)\n", memory_free, memory_free);
-    printf("There are %u non free blocks in the free list\n", non_free_blocks);
-    printf("There are %u inconsistent sized blocks in the free list\n", inconsistent);
+    printf("memory free in malloc chain: %m (%zu)\n", memory_free, memory_free);
+    printf("There are %zu non free blocks in the free list\n", non_free_blocks);
+    printf("There are %zu inconsistent sized blocks in the free list\n", inconsistent);
```

Comment:
While correct for standard printf, this is not correct here. I am using my own implementation of printf and I have not implemented zu yet.

Reply:
Done, sorry it took so long.